src/app/dashboard/contracts/components/ContractItem.jsx

```diff
@@ -0,0 +1,86 @@
+import Image from 'next/image'
+import { Accordion, Table } from 'flowbite-react'
+import { HiOutlineCurrencyEuro, HiCalendar, HiDotsHorizontal, HiCheck, HiExclamationCircle } from 'react-icons/hi'
+import mockContractTransfer from '@/utils/data/mockContractTransfers.json'
+
+function ContractItem ({ vc, price }) {
+  const maxLengthTitle = 42
+  const maxLengthDescription = 60
+  const name = vc.title.length > maxLengthTitle ? vc.title.substring(0, maxLengthTitle) + '...' : vc.title
+  const description = vc.short_description.length > maxLengthDescription ? vc.short_description.substring(0, maxLengthDescription) + '...' : vc.short_description
+  const issuanceDate = vc.created_at ? new Date(vc.created_at) : new Date()
+  const date = isNaN(issuanceDate.getTime()) ? new Date() : issuanceDate
+  const provider = vc.provider
+  const providedBy = provider.name ?? 'OTHER'
+  const validatedPrice = price ?? '0'
+  const history = mockContractTransfer.transfer_history
+  const historyData = ['', 'Status', 'Date', 'Transfer ID']
+
+  return (
+    <Accordion collapseAll flush className=' min-w-fit overflow-auto m-5 shadow-md rounded-md '>
+      <Accordion.Panel>
+        <Accordion.Title className='flex bg-white'>
+          <div className='flex'>
+            <div className='flex flex-col'>
+              <div className=' flex text-lg font-semibold'>{name}</div>
+              <div className='flex flex-col mt-4 w-96 min-w-40 max-w-96'>
+                <div className=' text-sm mr-4 w-2/3'>{description}</div>
+                {/* Price and date */}
+                <div className='flex flex-row items-end mt-3'>
+                  <div className='flex flex-row gap-2 w-36 mt-2'>
+                    <HiCalendar size={20} />
+                    <p className='text-sm'>{date.toISOString().split('T')[0]}</p>
+                  </div>
+                  <div className='flex flex-row gap-2 w-36'>
+                    <HiOutlineCurrencyEuro size={20} />
+                    <p className='text-sm'>{validatedPrice} euros</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className='flex gap-2 max-h-16'>
+              <Image width={64} height={64} src={provider.picture} alt={provider.name} className='max-w-16 max-h-16 object-cover object-center rounded-lg shadow-lg ml-2' />
+              <p className=' flex pr-2 text-sm font-semibold'>{providedBy}</p>
+            </div>
```

Comment:
I have tried to put it in the center or completely to the right.
Since this accordion component is a flowbite component it has inner elements that are not easy to override, I tried to create another custom theme for this specific case, but it still does not override the exact element that contains the title. There is always space to the right.
So in the end I have tried to put it to the right as much as I could, but if the VP is very wide it would look like it is in the center
![image](https://github.com/user-attachments/assets/d5624076-8a47-41e8-924e-0cf55d6e38e5)
![image](https://github.com/user-attachments/assets/c801b80b-8f4d-4cdb-829a-e7fb96eb25f0)